src/types/markers.js

```diff
@@ -418,7 +418,7 @@ export type NetworkStatus =
   | 'STATUS_STOP'
   | 'STATUS_REDIRECT'
   | 'STATUS_ABORT';
-export type NetworkRedirectType = 'permanent' | 'temporary' | 'internal';
+export type NetworkRedirectType = 'Permanent' | 'Temporary' | 'Internal';
```

Comment:
Now thinking about this, I wonder if we should have an enum-like value for them instead, like `NetworkStatus`. And then we can convert to their human readable versions in the front-end.

Reply:
We can do that, but is that better ? :-)
This is already an enum-like value, except I use it directly now, but in the future we can still provide different values if we want (or localize!).

Reply:
Yeah, probably we can still change this to use it like an enum value to make it localizible. It's okay for me then.